pkg/secretlock/api.go

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package secretlock
+
+// package secretlock contains secret lock services to secure keys used by the Aries agent
+// and more specifically used by the KMS service.
+
+// Service provides crypto service used internally by the KMS
```

Comment:
This could probably have more description.

Reply:
I will add more description for this service..